### DISCUSSION TITLE
Medpy library

### DISCUSSION THREAD

**Discussion by vijaypshah** [MIC-DKFZ/nnUNet#1319] [Q&A] [answered]
Hi,

Medpy library has GPLv3 license terms. Is it ok to use this library and combine it with nnUNet which is release as Apache 2.0 ?

Thank you,

Vijay

**Comment by FabianIsensee** [comment]
I am no lawyer but I would argue that only if I actually used/integrated code from the repository in nnU-Net then this would be a problem. I am using it via the API it provides and it is thus a clearly separated module and visible as such

**Comment by FabianIsensee** [comment, marked as answer]
medpy wasn't even used anymore so I removed it now

**Reply by vijaypshah** [reply]
Thank you! That's great!
